src/frontend/screens/Library/components/InstallModal/WineSelector/index.tsx

```diff
@@ -24,64 +27,86 @@ export default function WineSelector({
   winePrefix,
   wineVersionList,
   wineVersion,
-  appName,
-  runner
+  title = 'sideload'
 }: Props) {
   const { t } = useTranslation('gamepage')
 
+  const [useDefaultSettings, setUseDefaultSettings] = React.useState(true)
```

Comment:
A default value of `true` might be ideal for your specific setup right now, but most people have plenty of space for Prefixes & it's *really* the safer option to have one per game

Reply:
Do you think so? Would be good to have some input from the users someday because this is more speculation in my experience.
Also, for a UI point of view it makes the Install screen cleaner.

Reply:
A clean prefix could be 200MB but if you install some dependencies with winetricks that can go way up. I have prefixes here that are taking more than 2GB, and that is with btrfs compresion enabled.

I still think that we should test with it default first, is not like I am removing the option, the person always can go and click the checkbox if they want to have a separate prefix for each game.

Reply:
If you are knowledgeable enough to install dependencies via Winetricks, you are also knowledgeable enough to toggle this option yourself when installing games (and sidenote, if you're installing things with Winetricks, you *absolutely should use a separate prefix*). I'm not advocating against the option as a whole, I just have an issue with it being the default.
Most people are just going to go "Oh, why would I change the default settings" when in reality, unless you're in a specific situation (low on space), *not* checking that option is safer

Reply:
ok, mate. changed to false now.